### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN027.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#73] [General]
# https://blog.3place.cn/3place/lmyn/LMYN027.html

https://blog.3place.cn/3place/lmyn/LMYN027.html

**Comment by picklecai** [comment]
这是印度版的“十八般武器”列表。

**Comment by picklecai** [comment]
罗摩的身材特点，今天提到了长胳膊，这也是善于打斗的特征吧。

**Comment by picklecai** [comment]
纯洁的牟尼中的尊者，
面向东方，站在那里，
满怀喜悦地交给了罗摩
一大串至高无上的咒语。 （1.26.21）

这些是神兵，靠咒语出现的。
